### ISSUE TITLE
How to set iso alternatives with i18n

### ISSUE THREAD

**Issue by learntheropes** [harlan-zw/nuxt-simple-sitemap#106]
### Details

I have an i18n project with the following settings:

```js
  i18n: {
    locales: [
	  {
	    code: 'en',
	    file: 'en.json',
	    iso: 'en-US'
	  }, 
	  {
	    code: 'es',
	    file: 'es.json',
	    iso: 'es-419'
	  },
	  {
	    code: 'pt',
	    iso: 'pt-BR',
	    file: 'pt.json
	  }
	],
    defaultLocale: 'es',
    lazy: true,
    langDir: 'lang',
    strategy: 'prefix',
    detectBrowserLanguage: {
      useCookie: false,
      redirectOnRoot: true,
      redirectOn: 'root',
    }
  },
```

i18n creates in the page head alternatives for both the codes (en, es, pt) and for the iso (en-US, es-419, pt-BR).

The static routes of the sitemap are created only for the codes (en, es, pt).

How can I create them also for the iso (en-US, es-419, pt-BR)?

I tried both the following without any lucky.
```
autoalternativelangprefixes: ['en-US', 'es-419', 'pt-BR']
autoAlternativeLangPrefixes: true
```

**Comment by harlan-zw** [comment]
Hey, thanks for raising this.

As of [v3.1.3](https://github.com/harlan-zw/nuxt-simple-sitemap/releases/tag/v3.1.3), the i18n iso will be used over the code.

Also if you provide your own `autoalternativelangprefixes` it should now use those instead of replacing them.

Let me know f you have any further issues with it.

**Comment by learntheropes** [comment]
@harlan-zw This is only partially resolved by v3.1.3.

With `nuxi dev` I still have the codes and `x-dafault` has been added:
```xml
<url>
<loc>http://localhost:3000/pt</loc>
<xhtml:link rel="alternate" hreflang="en" href="http://localhost:3000/en"/>
<xhtml:link rel="alternate" hreflang="es" href="http://localhost:3000/es"/>
<xhtml:link rel="alternate" hreflang="pt" href="http://localhost:3000/pt"/>
<xhtml:link rel="alternate" hreflang="x-default" href="http://localhost:3000/es"/>
</url>
<url>
<loc>http://localhost:3000/en/blog</loc>
<xhtml:link rel="alternate" hreflang="en" href="http://localhost:3000/en/blog"/>
<xhtml:link rel="alternate" hreflang="es" href="http://localhost:3000/es/blog"/>
<xhtml:link rel="alternate" hreflang="pt" href="http://localhost:3000/pt/blog"/>
<xhtml:link rel="alternate" hreflang="x-default" href="http://localhost:3000/es/blog"/>
</url>
```

With `nuxi build` I have different situations:

For the index page, I now have all the codes and iso, but 2 x-default, of which one is wrong (should be `es`, I have `es` and `en`):
```xml
<url>
<loc>http://localhost:3000/en</loc>
<xhtml:link rel="alternate" hreflang="x-default" href="http://localhost:3000/en"/>
<xhtml:link rel="alternate" hreflang="en-US" href="http://localhost:3000/en-US/en"/>
<xhtml:link rel="alternate" hreflang="es-419" href="http://localhost:3000/es-419/en"/>
<xhtml:link rel="alternate" hreflang="pt-BR" href="http://localhost:3000/pt-BR/en"/>
<xhtml:link rel="alternate" hreflang="en" href="http://localhost:3000/en"/>
<xhtml:link rel="alternate" hreflang="es" href="http://localhost:3000/es"/>
<xhtml:link rel="alternate" hreflang="pt" href="http://localhost:3000/pt"/>
<xhtml:link rel="alternate" hreflang="x-default" href="http://localhost:3000/es"/>
</url>
```

For the other static routes, it applies the codes alternatives and the x-default but not the iso:
```xml
<url>
<loc>http://localhost:3000/en/blog</loc>
<xhtml:link rel="alternate" hreflang="en" href="http://localhost:3000/en/blog"/>
<xhtml:link rel="alternate" hreflang="es" href="http://localhost:3000/es/blog"/>
<xhtml:link rel="alternate" hreflang="pt" href="http://localhost:3000/pt/blog"/>
<xhtml:link rel="alternate" hreflang="x-default" href="http://localhost:3000/es/blog"/>
```

See a reproduction here: https://stackblitz.com/edit/nuxt-sitemap?file=nuxt.config.js&initialpath=sitemap.xml

I also tried using `[autoalternativelangprefixes](autoalternativelangprefixes: ['en-US', 'es-419', 'pt-BR'])` but it didn't fix the issue. Am I using it correctly?

I guess all the static routes should be consistent by default, and you may want to fix it.

Btw, thank you a lot for your work and for the quick fixes on the other issues!
